src/controllers/DynamicFacetQueryController.ts

```diff
@@ -108,10 +115,11 @@ export class DynamicFacetQueryController {
   }
 
   protected get currentValues(): IFacetRequestValue[] {
-    return this.facet.values.allFacetValues.map(({ value, state, preventAutoSelect }) => ({
+    return this.facet.values.allFacetValues.map(({ value, state }) => ({
       value,
       state,
-      preventAutoSelect
+      // TODO: remove after SEARCHAPI-4233 is completed
+      preventAutoSelect: this.preventAutoSelection
```

Comment:
What are the consequences if we don't do this ?

Does it just become a "superfluous" parameter, or is there are other consequences ?

Reply:
It becomes a double dip parameter, specifying it both on the facet and all the facet values will have the same behaviour